scripts/1d_sinusoidal_growth.py

```diff
@@ -0,0 +1,295 @@
+import argparse
```

Comment:
As the number of scripts grow, I think consistent naming will be helpful, and I think it's worth adding `_client` to the end of all client scripts.